### DISCUSSION TITLE
Add Grafana dashboard with the same queries as ClickHouse's Advanced Dashboard

### DISCUSSION THREAD

**Discussion by DerekChia** [grafana/clickhouse-datasource#886] [Ideas]
ClickHouse has a built-in advanced observability dashboard [1] that is included in the binary, showing a variety of metrics [2]. 

It would be good to have a separate pre-built Grafana dashboard in the plugin that displays the same metrics as the advanced dashboard. This would help centralize the monitoring efforts. 

[1] https://clickhouse.com/docs/en/operations/monitoring
[2] https://clickhouse.com/docs/en/operations/system-tables/dashboards

**Comment by SpencerTorres** [comment]
Thanks!

Solved in https://github.com/grafana/clickhouse-datasource/pull/923
